### DISCUSSION TITLE
integrate with multi-tenants?

### DISCUSSION THREAD

**Discussion by kertahayu** [freekmurze/laravel-analytics#385] [Q&A]
I've seen the documentation. my question is how to implement it with multi-tenant applications?

all users manage their application on `dashboard.foo.com`.
each user has their own application `user.bar.com`.

exmaple :
user a have app `a.bar.com`
user b have app `b.bar.com`

how to dynamically create properties in google analytic?
how do i implement google analytics by identifying each user?

**Comment by brodsolutions** [comment]
Great Question. Can you feed in more than one Google Analytics?

**Reply by kertahayu** [reply]
I only use google analytics on the web, that's why I don't understand whether google analytic can handle that.

**Comment by Omycron-devs** [comment]
yes i need to retrieve stats for thousands of sites too, all have their own viewid/json

**Reply by kertahayu** [reply]
it is very difficult to manage. especially if the dynamic application is like I mean above. Hehehe
